etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/OrderController.java

```diff
@@ -20,7 +20,7 @@ public class OrderController {
     @Inject HapiFhir fhir;
     @Inject Logger logger;
     @Inject MetricMetadata metadata;
-    @Inject RuleEngine ruleEngine;
+    @Inject ValidationRuleEngine validationEngine;
```

Comment:
You should be injecting the RulesEngine, not the implementation of the interface

Reply:
We need to inject two different rules engine, one for the validation and another one for the transformation. The transformation engine is being implemented in the other active PR that we have